joint_trajectory_controller/include/joint_trajectory_controller/hardware_interface_adapter.h

```diff
@@ -213,7 +216,7 @@ class HardwareInterfaceAdapter<hardware_interface::VelocityJointInterface, State
     // Update PIDs
     for (unsigned int i = 0; i < n_joints; ++i)
     {
-      const double command = pids_[i]->computeCommand(state_error.position[i], state_error.velocity[i], period);
+      const double command = (desired_state.velocity[i] * use_velocity_ff_) + pids_[i]->computeCommand(state_error.position[i], state_error.velocity[i], period);
```

Comment:
I am not sure if that really made sense, but we used 0.9 for the Care-O-bot 3 arms (pre-`ros_control` stack).
I don't see a reason to restrict it to either `0.0` or `1.0`, but I am no contoller expert.